satellite/contact/db_test.go

```diff
@@ -0,0 +1,170 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package contact_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"storj.io/storj/internal/testcontext"
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/storj"
+	"storj.io/storj/satellite"
+	"storj.io/storj/satellite/overlay"
+	"storj.io/storj/satellite/satellitedb/satellitedbtest"
+)
+
+func TestUpdateCheckIn(t *testing.T) {
```

Comment:
Awesome test. Would you consider moving this into `satellite/overlay/service_test.go` since this is testing the overlay method rather than contact endpoint or service?